ocp_resources/datavolume.py

```diff
@@ -339,8 +326,5 @@ def delete(self, wait=False, timeout=TIMEOUT_4MINUTES, body=None):
         Returns:
             bool: True if delete succeeded, False otherwise.
         """
-        # if garbage collector is enabled, DV will be deleted after success
         if self.exists:
             return super().delete(wait=wait, timeout=timeout, body=body)
```

Comment:
as the logic is removed from `delete`, this function can be deleted

Reply:
for now I added a deprecation warning 
GC didnt removed from main branch yet